src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,67 @@
+package calculator;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class StringAddCalculator {
+    private static final Pattern CUSTOM_DELIMITER_PATTERN = Pattern.compile("^//(.)\\n(.*)$");
+    private static final String COMMA_OR_COLON_DELIMITER = "[,:]";
+    private static final int NULL_AND_BLANK_VALUE = 0;
+    private static final int MATCHER_CUSTOM_DELIMITER = 1;
+    private static final int MATCHER_TEXT = 2;
+
+    public static int splitAndSum(String text) {
+        if(checkNullAndBlank(text)) {
+            return NULL_AND_BLANK_VALUE;
+        }
+
+        Matcher matcher = CUSTOM_DELIMITER_PATTERN.matcher(text);
+        if(matcher.find()) {
+            return sum(toIntegerList(splitDelimiter(
+                    matcher.group(MATCHER_CUSTOM_DELIMITER),matcher.group(MATCHER_TEXT))));
```

Comment:
```suggestion
            String group = matcher.group(MATCHER_CUSTOM_DELIMITER);
            return sum(toIntegerList(splitDelimiter(group, matcher.group(MATCHER_TEXT))));
```
이런식으로 정리해보면 어떨까요?
지금은 코드를 읽기가 어려운것 같아요. 😭

Reply:
메서드 체이닝과 상수 네이밍이 길어지다 보니
가독성이 어려운 단점이 보였던 거 같습니다.
항상 이런 부분이 쉽지 않더라구요!
수정하겠습니다.